contrib/initramfs/scripts/zfs.in

```diff
@@ -424,6 +424,34 @@ decrypt_fs()
 			if ! [ "${KEYLOCATION}" = "prompt" ]; then
 				$ZFS load-key "${ENCRYPTIONROOT}"
 
+                        # Wait for input from named pipe, if Dropbear is running
+			elif [ ! -z "$(ps | grep dropbear | grep -v grep 2> /dev/null)" ]; then
```

Comment:
If you turn one of the characters into a single-character class, you can avoid the need for the second grep, as the pattern will not match itself: `ps | grep [d]ropbear`